app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<main>
```

Comment:
- [ ] @Team great work on your app, however, I was unable to move past the login or registration page, I successfully submit my details but it doesn't log me in kindly take  a look at it, here is a snapshot of users created using the form 
![image](https://user-images.githubusercontent.com/61408860/192832136-5a3e1e62-b5ab-468e-b64d-404af8758f4e.png)

Reply:
Hello @danmainah. It works just fine to me

I don't know what can be the problem in this case 😅 

https://user-images.githubusercontent.com/86022690/192836952-da7c2af4-e6f9-4373-a0a5-f776a882bffc.mov

Reply:
@danmainah  Sorry for the video quality but it was for making it less than 10 MB